Update to Highline 2.1.0

kbrock opened the pull request:
steal anything you want out of here @bdunne 

not thrilled with the changes to tests with the expect heard stuff.
wish the questions were not going through, but this is the least invasive way I could think to solve

overlaps with #200 

Fixes #181

Fryguy commented on spec/database_admin_spec.rb:
```diff
@@ -102,7 +102,7 @@
 
           error = "Please provide #{errmsg}"
-          expect_heard ["Enter the #{prmpt}: ", error, prompt]
+          expect_heard ["Enter the #{prmpt}: |#{"/tmp/evm_db.backup"}| ", error, prompt]
```
```suggestion
          expect_heard ["Enter the #{prmpt}: |/tmp/evm_db.backup| ", error, prompt]
```

kbrock commented:
update:
- fixed highline gem value
- removed string interpolation in test
- moved dependency to spot where dependencies were defined (it was not finding the files for some reason, but it may have been another problem. seemed good to me anyway)

I tested on a nightly appliance. had to muck with the appliance console gemspec in 2 places and the vmdb/gemfile.lock, but I was able to setup dhcp, create a v2 key, set the region and configure an internal database.

kbrock commented:
@bdunne let me know if you are happy with these changes.
as I mentioned I was able to change dhcp, and setup a basic appliance. If there are other configuration options you want me to test, let me know.

kbrock commented:
@Fryguy or @bdunne I did run through those few tests. Anything else we want to do to be comfortable merging?

Fryguy reviewed (approved):
LGTM - @bdunne Please also review.

Fryguy commented:
@kbrock Can you run the full rspec suite locally?  We still don't have github actions working in this repository yet.

Fryguy commented:
Bouncing now that GitHub Actions is in place - 3.0 should still fail cause of kwargs at the moment

kbrock commented:
update:
- removed the pending
update:
- removed the CI tty hack (still needed)
update
- testing out alternative no-echo input

Fryguy reviewed (approved):
LGTM

Fryguy commented:
Love that you could kill the tty hack too
